content/influxdb/cloud/write-data/troubleshoot.md

```diff
@@ -74,13 +77,62 @@ If you notice data is missing in your bucket, do the following:
 - Verify the timestamps match the [precision parameter](/influxdb/cloud/write-data/#timestamp-precision).
 - Minimize payload size and network errors by [optimizing writes](/influxdb/cloud/write-data/best-practices/optimize-writes/)
 
-### Troubleshoot rejected points
-
-InfluxDB may reject points even if the HTTP request returns "Success".
-If some of your data did not write to the bucket, check for [field type](/influxdb/cloud/reference/key-concepts/data-elements/#field-value) differences between the missing data point and other points that have the same [series](/influxdb/cloud/reference/key-concepts/data-elements/#series).
-For example, did you attempt to write `string` data to an `int` field?
-
-InfluxDB rejects points for the following reasons:
-- The **batch** contains another point with the same series, but one of the fields has a different value type.
-- The **bucket** contains a saved point with the same series, but one of the fields has a different value type.
-
+### Review rejected points
+
+InfluxDB may have rejected points even if the HTTP request returned "Success".
+To get a log of rejected data points, query the [`rejected_points` measurement](/influxdb/cloud/reference/internals/system-buckets/#_monitoring-bucket-schema) in your organization's `_monitoring` bucket.
+
+```js
+from(bucket: "_monitoring")
+  |> range(start:-1h)
+  |> filter(fn:(r) =>
+    r._measurement == "rejected_points"
+  )
+```
```

Comment:
It might be worth selecting on the field `count` here, as some rejected points errors (parsing errors in particular) also add an `error` measurement with some textual information about the failure.